python/ray/experimental/state/common.py

```diff
@@ -356,10 +360,15 @@ class RuntimeEnvState(StateSchema):
 
 @dataclass(init=True)
 class ListApiResponse:
-    # Total number of the resource from the cluster.
-    # Note that this value can be larger than `result`
-    # because `result` can be truncated.
+    # Total number of the available resource from the cluster.
```

Comment:
Can you have a docstring to explain the relationships between these values?

total -> total resources
num_after_truncation -> # of entries after truncation from the source (for system stability)
num_filtered -> # of entries for the query
len(data) -> # of entries after limiting